graviti/schema/builtin.py

```diff
@@ -261,29 +261,14 @@ class record(PortexBuiltinType):  # pylint: disable=invalid-name
           },
         )
 
-        Create a record by :class:`Field` list:
-
-        >>> t = record([Field("f0", float64(0)), Field("f1", array(int64()))])
```

Comment:
Field is also tuple, it is legal input of OrderDict, shall we put the Field remove job into another PR?